onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -607,7 +607,32 @@ std::unique_ptr<IExecutionProvider> CreateExecutionProviderInstance(
 #endif
   } else if (type == kDnnlExecutionProvider) {
 #ifdef USE_DNNL
-    return onnxruntime::DnnlProviderFactoryCreator::Create(session_options.enable_cpu_mem_arena)->CreateProvider();
+    // Generate dnnl_options
+    OrtDnnlProviderOptions dnnl_options;
+// For Eigen and OpenMP
+#if !defined(DNNL_ORT_THREAD)
```

Comment:
Not sure where this is defined.

Reply:
Was a remnant of previous PR, updating it now